phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/installations/InstallationsView.java

```diff
@@ -87,6 +89,18 @@ public InstallationsView(ThemeManager themeManager, JavaFxSettingsManager javaFx
 
         this.activeInstallations = createInstallationListWidget();
 
+        // a binding containing the number of currently active installations
+        final StringBinding openInstallations = Bindings.createStringBinding(
+                () -> Integer.toString(this.activeInstallations.getElements().size()),
```

Comment:
What will happen if there are >9 elements? Can we maybe show a "+" or "i" instead of 10? I assume that more than one character will not fit in the circle.